modules/meta/arrow-meta-prototype/compiler-plugin/src/main/kotlin/arrow/meta/plugins/higherkind/HigherKindPlugin.kt

```diff
@@ -75,7 +75,7 @@ private val KtClass.partialKindAritySuffix: String
   get() = (arity - 1).let { if (it > 1) "$it" else "" }
 
 fun isHigherKindedType(ktClass: KtClass): Boolean =
-  ktClass.annotationEntries.any { it.text == "@higherkind" } &&
+  ktClass.annotationEntries.any { it.text.matches(Regex("@(arrow.)?higherkind")) } &&
```

Comment:
`\.`?

Reply:
Good point, @pakoito!! Thanks!! Solution: `\\.`

```
>>> val regex = Regex("@(arrow.)?higherkind")
>>> "@higherkind".matches(regex)
res1: kotlin.Boolean = true
>>> "@higherkind2".matches(regex)
res2: kotlin.Boolean = false
>>>  "@.higherkind".matches(regex)
res3: kotlin.Boolean = false
>>> "@arrow.higherkind".matches(regex)
res4: kotlin.Boolean = true
>>> "@arrow-higherkind".matches(regex)
res5: kotlin.Boolean = true
>>> 
```

``` 
>>> val regex = Regex("@(arrow\\.)?higherkind")
>>> "@higherkind".matches(regex)
res1: kotlin.Boolean = true
>>> "@higherkind2".matches(regex)
res2: kotlin.Boolean = false
>>> "@.higherkind".matches(regex)
res3: kotlin.Boolean = false
>>> "@arrow.higherkind".matches(regex)
res4: kotlin.Boolean = true
>>> "@arrow-higherkind".matches(regex)
res5: kotlin.Boolean = false
>>> 
```

Reply:
Fixed!! 
However, the target of this PR isn't `master`. 
I will be happy with your approval :wink:

Reply:
Go for it!